android/app/src/main/java/org/mozilla/magnet/ScannerBle.java

```diff
@@ -66,6 +66,11 @@ public void start() {
     @ReactMethod
     public void stop() {
         Log.d(TAG, "stop");
+        if (bluetoothAdapter == null || !bluetoothAdapter.isEnabled()) {
+            // Device does not support bluetooth, or is not enabled
+            return;
+        }
+
```

Comment:
It might be cleaner to rely on the single check in `.start()` and set a `started` flag that we check for inside `.stop()`.